servers/rendering/renderer_rd/environment/fog.cpp

```diff
@@ -543,7 +543,7 @@ void Fog::volumetric_fog_update(const VolumetricFogSettings &p_settings, const P
 		if (p_cam_projection.is_orthogonal()) {
 			fog_near_size = fog_far_size;
 		} else {
-			fog_near_size = Vector2();
+			fog_near_size = Vector2(MAX(0.001f, frustum_near_size.x), MAX(0.001f, frustum_near_size.y));
```

Comment:
```suggestion
			fog_near_size = frustum_near_size.max(Vector2(0.001, 0.001));
```
Convenience, also use double precision constants for cases with `real_t`